addon/components/listbox/-option.hbs

```diff
@@ -0,0 +1,26 @@
+{{#let (element (or @as 'li')) as |Tag|}}
+  <Tag
+    role='option'
+    id={{this.guid}}
+    tabindex='-1'
+    {{did-insert (fn @registerOptionElement this)}}
+    {{on 'focus' (fn @setActiveOption this)}}
+    {{on 'mouseover' (fn @setActiveOption this)}}
+    {{on 'mouseout' @unsetActiveOption}}
+    {{on 'click' this.handleClick}}
+    aria-selected={{if
+      this.isSelectedOption
+      (if this.isSelectedOption 'true' 'false')
+    }}
```

Comment:
I'm curious about this conditional; it seems like the only values it will produce are `"true"` or `undefined`. How does `"false"` actually end up returned from this expression?

If `isSelectedOption` is truthy, then both conditionals evaluate to `true` and `"true"` is the result

If `isSelectedOption` is false, then the first conditional will return `undefined` since there isn't an "else" condition in there.

I _think_ the conditional on like 13 here will _always_ return `true` since it'll only even be evaluated if the `this.isSelectedOption` is truthy

Reply:
You're right, that makes no sense 🤦 

https://github.com/GavinJoyce/ember-headlessui/pull/76/commits/bdce10eeeb8dd5c1ee2ef9fe0c6d2a1cc9ffb750 has a simplified conditional which evaluates to `aria-selected="true"` if the option is selected, or omits the attribute if not selected.